pydm/data_plugins/plugin.py

```diff
@@ -113,6 +115,13 @@ def receive_from_channel(self, payload):
         pass
 
     def send_to_channel(self):
+        time = datetime.datetime.now()
+        if self.last_notify is None:
+            diff_time = 1
+        else:
+            diff_time = (time - self.last_notify).total_seconds()
+        self.notify_freq = round(1.0/diff_time, 0)
```

Comment:
Maybe something like:
```suggestion
        self.notify_freq = round(1.0 / max(1e-6, diff_time), 0)
```

Reply:
Thanks!